packages/fast-usdc/test/exos/advancer.test.ts

```diff
@@ -8,7 +8,7 @@ import type { Zone } from '@agoric/zone';
 import type { VowTools } from '@agoric/vow';
 import { prepareAdvancer } from '../../src/exos/advancer.js';
 import { prepareStatusManager } from '../../src/exos/status-manager.js';
-import { prepareTransactionFeedKit } from '../../src/exos/transaction-feed.js';
+import { prepareTransactionFeedKit } from '../../src/exos/transaction-feed.ts';
```

Comment:
It looks like the build step does just a bit more than erase types.

I guess bundleSource groks, though